src/cli/commands/new.command.ts

```diff
@@ -33,7 +30,9 @@ export class NewCommand {
 		private readonly npmRunner: NpmRunner = new NpmRunner(),
 		private readonly bashRunner: BashRunner = new BashRunner(),
 		private readonly bashRunnerHusky: BashRunnerHusky = new BashRunnerHusky(),
-	) {}
+	) {
+		super();
+	}
 
 	public async execute() {
 		const printSuccess = Printer.format({fontColor: 'green', decoration: 'bold'});
```

Comment:
We don't need this anymore, right? We could use the parent's

Reply:
Yes! But I don't want to change it because it will have conflicts with the changes made by @dustytrinkets in another PR